src/Tizen.NET.MaterialComponents/MaterialComponents.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public enum ComponentsOptions
+    {
+        All,
+        Supported
+    }
+
+    public static class MaterialComponents
```

Comment:
What about deprecating `ThemeLoader` and adding B `MaterialComponents` instead?

Reply:
Fixed.